src/app/components/Game.tsx

```diff
@@ -27,6 +28,50 @@ const Game: FC = () => {
     'cake',
   ];
 
+  const drawnImage = '{"lines":[{"points":[{"x":203,"y":59.98333740234376},{"x":203,"y":59.98333740234376},{"x":182,"y":72.98333740234375},{"x":150,"y":101.98333740234375},{"x":131,"y":122.98333740234375},{"x":74.99999999999997,"y":299.98333740234375},{"x":101,"y":337.98333740234375},{"x":137,"y":356.98333740234375},{"x":254,"y":358.98333740234375},{"x":266,"y":353.98333740234375},{"x":277,"y":347.98333740234375},{"x":303,"y":323.98333740234375},{"x":309,"y":315.98333740234375},{"x":312,"y":310.98333740234375},{"x":315,"y":286.98333740234375},{"x":313,"y":276.98333740234375},{"x":310,"y":267.98333740234375},{"x":287,"y":217.98333740234375},{"x":279,"y":206.98333740234375},{"x":269,"y":200.98333740234375},{"x":264,"y":198.98333740234375},{"x":220,"y":219.98333740234372},{"x":210,"y":234.98333740234375},{"x":203,"y":247.98333740234375},{"x":191,"y":285.98333740234375},{"x":191,"y":287.98333740234375},{"x":211,"y":273.98333740234375},{"x":218,"y":257.98333740234375},{"x":219,"y":252.98333740234375},{"x":215,"y":192.98333740234375},{"x":208,"y":182.98333740234375},{"x":172,"y":128.98333740234375},{"x":168,"y":122.98333740234375},{"x":164,"y":118.98333740234375},{"x":155,"y":108.98333740234375},{"x":153,"y":110.98333740234375},{"x":153,"y":111.98333740234375}],"brushColor":"#444","brushRadius":10}],"width":400,"height":400}';
```

Comment:
Dette er mocks som kun brukes for testing,right? Kunne vi ha flyttet det til en egen fil?

Reply:
Yup